scripts/docs/snippetadapter.mjs

```diff
@@ -8,6 +8,9 @@ import { constants, readFile, writeFile, copyFile, access, mkdir } from 'node:fs
 import upath from 'upath';
 import { build as esbuild } from 'esbuild';
 import { CKEDITOR5_COMMERCIAL_PATH, CKEDITOR5_ROOT_PATH } from '../constants.mjs';
+import umbertoJson from '../../docs/umberto.json' with { type: 'json' };
+
+const CKBOX_VERSION = umbertoJson.variables.CKBOX_VERSION;
```

Comment:
```suggestion
const { CKBOX_VERSION } = umbertoJson.variables;
```

- Cosmetic, but in this case, enough to shorten it.
- I do not see any confirmation that the built documentation loads a proper resource.
- There are three (or even four, soon) commits. Can we merge all-in?

Reply:
I've built docs, visited https://172.24.224.54:8080/ckeditor5/latest/features/file-management/ckbox.html. CKBox demo works, and `CKBox.version` in console returns `"2.9.2"`